feat: translate guides/accessibility/learning-resources.md (#159)
DazzlingFugu/ember-fr-guides-source: guides/release/accessibility/learning-resources.md

> Ces ressources d'apprentissage sur l'accessibilité ont pour but d'apporter une aide additionnelle aux développeurs.euses pour améliorer la qualité de leur code.

Reviewer: As discussed on Discord
```suggestion
Ces ressources d'apprentissage sur l'accessibilité ont pour but d'apporter une aide additionnelle aux développeurs et développeuses pour améliorer la qualité de leur code.
```

> *Note: Les liens ci-dessous redirigent vers des articles écrits en Anglais

Reviewer: ```suggestion
*Note: La plupart des liens ci-dessous redirigent vers des articles écrits en Anglais
```

- [Documentation Web MDN: Accessibilité](https://developer.mozilla.org/en-US/docs/Learn/Accessibility)
- [Utiliser ARIA:](https://www.w3.org/TR/using-aria/) un guide pratique pour développeurs.euses: Comment ajouter des informations d'accessibilité sur des éléments HTML

> - [Web Content Accessibility Guidelines(WCAG): Directives du contenu web sur l'accessibilité 2.1](https://www.w3.org/TR/WCAG21/)

Author: Do we want to keep some terms in English? Here I kept Web Content A11y Guidelines as it's the abbreviation of WCAG

Reviewer: Answered in the suggestions 👍

Author: Thank you for the thorough review!

- [Accessible Rich Internet Applications (ARIA): Applications internet riches et accessibles 1.1](https://www.w3.org/TR/wai-aria-1.1/)
- [Comment répondre aux directives WCAG (référence rapide)](https://www.w3.org/WAI/WCAG21/quickref/)

## Ressources pratiques

- [Aperçus sur l'accessibilité](https://accessibilityinsights.io/)- Utiliser FastPass pour trouver des problèmes communs à fort impacts
- [extension aXe pour Chrome](https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd)

> - [Aide sur l'accessibilité](https://a11ysupport.io/)- Comprendre si son code marche avec les technologies d'assistance
> - [Où et comment ajouter un bug d'accessibilité](https://www.digitala11y.com/how-where-to-report-accessibility-bugs/)

Reviewer: ```suggestion
- [Accessibility Support](https://a11ysupport.io/)- Comprendre si son code marche avec les technologies d'assistance
- [How and where to report accessibility bugs](https://www.digitala11y.com/how-where-to-report-accessibility-bugs/)
```

### Design

> - [ColorSafe](http://colorsafe.co/)- Aider les designers avec de magnifiques palettes de couleurs accessibles, basées sur les directives WCAG, ratio de couleurs de contraste de textes et arrières plans.

Reviewer: I suggest "Appuyer" which corresponds maybe a bit more to "Empowering" while "Aider" is more about solving an issue they couldn't solve themselves. The sentence makes a bit more sense with "avec ratio ..." so I would also replace the first occurrence to avoid repetition.

```suggestion
- [ColorSafe](http://colorsafe.co/)- Appuyer les designers en proposant de magnifiques palettes de couleurs accessibles, basées sur les directives WCAG, avec ratio de couleurs de contraste de textes et arrières plans.
```

> - [Créateur de palettes de couleur accessibles](https://toolness.github.io/accessible-color-matrix/)

Reviewer: As this one is not properly an article, but a tool (just like the Chrome extension), I would keep it as you did here 👍

### Ecriture

> - [Langage clair et simple](https://plainlanguage.gov/) - Apprendre comment écrire d'une manière accessible et inclusive pour une compréhension de lecture plus facile.

Reviewer: 👇 In that case I would replace EmberGuides invented title with the actual web site title.
```suggestion
- [plainlanguage.gov](https://plainlanguage.gov/) - Apprendre comment écrire d'une manière accessible et inclusive pour une compréhension de lecture plus facile.
```

Reviewer 2: What about adding the french government page about accessibility to this list of links https://www.gouvernement.fr/accessibilite ?

Author: I tried to find official resources, but I think I don't know what term/word we use in French as the equivalent of plain language 😬 Do you know?

Reviewer 3: I was not able to find official resources either. As always we are lacking official resources about a11y :grim

Author: :(

### Autres articles utiles

> - [La différence entre la navigation clavier et lecteur d'écran](https://tink.uk/the-difference-between-keyboard-and-screen-reader-navigation/)

Reviewer: ```suggestion
- [The difference between keyboard and screen reader navigation](https://tink.uk/the-difference-between-keyboard-and-screen-reader-navigation/)
```

- [How to Meet WCAG (Quick Reference)](https://www.w3.org/WAI/WCAG21/quickref/)

## Ressources pratiques

- [Accessibility Insights](https://accessibilityinsights.io/)- Utiliser FastPass pour trouver des problèmes communs à fort impact
- [Extension aXe pour Chrome](https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd)
- [Accessibility Support](https://a11ysupport.io/)- Comprendre si son code marche avec les technologies d'assistance
- [How and where to report accessibility bugs](https://www.digitala11y.com/how-where-to-report-accessibility-bugs/)

### Design

- [ColorSafe](http://colorsafe.co/)- Appuyer les designers en proposant de magnifiques palettes de couleurs accessibles, basées sur les directives WCAG, avec ratio de couleurs de contraste de textes et arrières plans.
- [Créateur de palettes de couleur accessibles](https://toolness.github.io/accessible-color-matrix/)

### Écriture

- [Langage clair et simple](https://plainlanguage.gov/) - Apprendre comment écrire d'une manière accessible et inclusive pour une compréhension de lecture plus facile.

### Autres articles utiles

> - [The difference between keyboard and screen reader navigation](https://tink.uk/the-difference-between-keyboard-and-screen-reader-navigation/)

Reviewer 3: You forgot to translate this 😅 .

```suggestion
- [La différence entre navigation au clavier et navigation avec un lecteur d'écran](https://tink.uk/the-difference-between-keyboard-and-screen-reader-navigation/)
```

Author: @BlueCutOfficial Actually suggested to keep the name in English, so I applied her commit 😅 Let me know @KamiKillertO or Blue what is best!

Reviewer 3: Ok. It's fine like this 👍
